test/util/rpcauth-test.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test share/rpcauth/rpcauth.py
+"""
+import base64
+try:
+    import configparser
+except ImportError:
```

Comment:
I think this is unnecessary - this is for python2 compatibility. We now no longer support python2 across the project.

Reply:
Done